Dockerfile

```diff
@@ -0,0 +1,21 @@
+## -*- docker-image-name: "terraform-zendesk" -*-
```

Comment:
do we need to update the travis file to push this up to dockerhub?

Reply:
We don’t need travis to build image because DockerHub detects new commits to GitHub by webhook and starts build job on its self.

Reply:
Could that be disabled and moved into travis?

my only concern with that is that the full test suite (i.e the acceptance tests that communicate with the zendesk api) prior to checking into master, so it is feasible that a given commit in master could be broken much like what happened with renovate earlier this week.

IMO it would be safer if we waited until after the acceptance suite has ran before we push the image to the docker registry.

Reply:
This is current settings.

[![Image from Gyazo](https://i.gyazo.com/786a6f94b474779d99a80930331cb939.png)](https://gyazo.com/786a6f94b474779d99a80930331cb939)

I think travis would have too heavy work if we put the build job to it. Maybe `docker build` will take almost same time as acceptance test.

> it would be safer if we waited until after the acceptance suite has ran before we push the image to the docker registry.

We can use **Build trigger** instead of **Autobuild**. I think that it would resolve this problem
by sending POST request to the endpoint from Travis after acc test passed.

[![Image from Gyazo](https://i.gyazo.com/e22169793c2308807f84a43792d8583b.png)](https://gyazo.com/e22169793c2308807f84a43792d8583b)

Reply:
Please check bab9a13.
I turned Autobuild off and registered trigger URL to `$DOCKERHUB_TRIGGER_ENDPOINT`.